go/host/container/host_container.go

```diff
@@ -2,93 +2,49 @@ package container
 
 import (
 	"fmt"
-	"time"
 
-	"github.com/ethereum/go-ethereum/rpc"
-	"github.com/obscuronet/go-obscuro/go/common"
 	"github.com/obscuronet/go-obscuro/go/common/log"
 	"github.com/obscuronet/go-obscuro/go/common/metrics"
 	"github.com/obscuronet/go-obscuro/go/config"
-	"github.com/obscuronet/go-obscuro/go/ethadapter"
-	"github.com/obscuronet/go-obscuro/go/ethadapter/mgmtcontractlib"
 	"github.com/obscuronet/go-obscuro/go/host"
+	"github.com/obscuronet/go-obscuro/go/host/enclave"
+	"github.com/obscuronet/go-obscuro/go/host/events"
+	"github.com/obscuronet/go-obscuro/go/host/l1"
+	"github.com/obscuronet/go-obscuro/go/host/l2"
 	"github.com/obscuronet/go-obscuro/go/host/p2p"
-	"github.com/obscuronet/go-obscuro/go/host/rpc/clientapi"
 	"github.com/obscuronet/go-obscuro/go/host/rpc/clientrpc"
-	"github.com/obscuronet/go-obscuro/go/host/rpc/enclaverpc"
 	"github.com/obscuronet/go-obscuro/go/wallet"
 
 	gethlog "github.com/ethereum/go-ethereum/log"
 	hostcommon "github.com/obscuronet/go-obscuro/go/common/host"
 )
 
-const (
-	APIVersion1             = "1.0"
-	APINamespaceObscuro     = "obscuro"
-	APINamespaceEth         = "eth"
-	APINamespaceObscuroScan = "obscuroscan"
-	APINamespaceScan        = "scan"
-	APINamespaceNetwork     = "net"
-	APINamespaceTest        = "test"
-	APINamespaceDebug       = "debug"
-)
-
 type HostContainer struct {
-	host           hostcommon.Host
-	logger         gethlog.Logger
-	metricsService *metrics.Service
-	rpcServer      clientrpc.Server
+	// Host and Logger are temporarily public to allow fudge in the test wiring
+	// todo (@matt) make these private again once we no longer need to wire the Stop method into the host factories
+	Host   hostcommon.Host
+	Logger gethlog.Logger
 }
 
 func (h *HostContainer) Start() error {
-	h.metricsService.Start()
-
 	// make sure the rpc server has a host to render requests
-	err := h.host.Start()
+	err := h.Host.Start()
```

Comment:
This kinda defeats the purpose of the container.

It's just a passtrhough now because it has no underlying logic.